securedrop/tests/test_unit_source.py

```diff
@@ -80,90 +82,90 @@ def test_generate_has_login_link(self):
         # Should redirect to /lookup
-        self.assertEqual(rv.status_code, 200)
-        self.assertIn("because you are already logged in.", rv.data)
+        self.assertEqual(resp.status_code, 200)
+        self.assertIn("because you are already logged in.", resp.data)
 
     def test_create(self):
         with self.client as c:
-            rv = c.get('/generate')
+            resp = c.get('/generate')
             codename = session['codename']
-            rv = c.post('/create', follow_redirects=True)
+            resp = c.post('/create', follow_redirects=True)
             self.assertTrue(session['logged_in'])
             # should be redirected to /lookup
-            self.assertIn("Submit documents and messages", rv.data)
+            self.assertIn("Submit documents and messages", resp.data)
 
     def _new_codename(self):
-        return common.new_codename(self.client, session)
+        return TestSource.new_codename(self.client, session)
 
     def test_lookup(self):
-        """Test various elements on the /lookup page"""
+        """Test various elements on the /lookup page."""
         codename = self._new_codename()
-        rv = self.client.post('login', data=dict(codename=codename),
-                              follow_redirects=True)
+        resp = self.client.post('login', data=dict(codename=codename),
+                                follow_redirects=True)
         # redirects to /lookup
-        self.assertIn("public key", rv.data)
+        self.assertIn("public key", resp.data)
         # download the public key
-        rv = self.client.get('journalist-key')
-        self.assertIn("BEGIN PGP PUBLIC KEY BLOCK", rv.data)
+        resp = self.client.get('journalist-key')
+        self.assertIn("BEGIN PGP PUBLIC KEY BLOCK", resp.data)
 
     def test_login_and_logout(self):
-        rv = self.client.get('/login')
-        self.assertEqual(rv.status_code, 200)
-        self.assertIn("Login to check for responses", rv.data)
+        resp = self.client.get('/login')
+        self.assertEqual(resp.status_code, 200)
+        self.assertIn("Login to check for responses", resp.data)
 
         codename = self._new_codename()
         with self.client as c:
-            rv = c.post('/login', data=dict(codename=codename),
+            resp = c.post('/login', data=dict(codename=codename),
                         follow_redirects=True)
-            self.assertEqual(rv.status_code, 200)
-            self.assertIn("Submit documents and messages", rv.data)
+            self.assertEqual(resp.status_code, 200)
+            self.assertIn("Submit documents and messages", resp.data)
             self.assertTrue(session['logged_in'])
-            common.logout(c)
+            resp = c.get('/logout', follow_redirects=True)
 
         with self.client as c:
-            rv = self.client.post('/login', data=dict(codename='invalid'),
+            resp = self.client.post('/login', data=dict(codename='invalid'),
```

Comment:
I think it works either way, but I can change it because it's not a sensible block.